### PR TITLE
Repo sync for protected CLA branch

### PR DESCRIPTION
**Pull request by opbld15**
The pull request is created from public637915981872726566 to public to fix git push error for protected CLA branch

### CODE REVIEW

**Comment by officedocspr4** [comment]
## STOP  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **69** files, which exceeds the **warning** limit of **69**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

Label UI screenshot

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

File: microsoft-365/business-premium/m365bp-device-groups-mdb.md [hunk 1]
```diff
@@ -66,7 +65,56 @@ You can create a new device group while you are in the process of creating or ed
...
+
+3. On the **Create device category** pane, enter a name for the new category, and an optional description.
+
+4. When you're done, select **Create**. You can see the new category in the list.
+
+Use the device category name when you create the Azure Active Directory (Azure AD) security groups. When users enroll their devices, they are presented with a list of the categories you configured in Intune. After they choose a category and finish enrollment, their device is added to the Active Directory security group that is associated with it.
+
+## Create dynamic device groups in Azure Active Directory
+
+You can also enter the Azure Active Directory (Azure AD) portal ([https://portal.azure.com](https://portal.azure.com)) from the Microsoft 365 admin center. In the Microsoft 365 admin center ([https://admin.microsoft.com](https://admin.microsoft.com/)), choose **All admin centers**, and then choose **Azure Active Directory**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-device-groups-mdb.md:84]:
```suggestion
You can also enter the [Azure AD portal](https://portal.azure.com) from the Microsoft 365 admin center. In the [Microsoft 365 admin center](https://admin.microsoft.com/), choose **All admin centers**, and then choose **Azure Active Directory**.
```

File: microsoft-365/business-premium/m365bp-device-groups-mdb.md [hunk 2]
```diff
@@ -66,7 +65,56 @@ You can create a new device group while you are in the process of creating or ed
...
+## After device groups are created
 
-Now that you've completed your primary missions, set up your [response teams](m365bp-security-incident-management.md) and [maintain your environment](m365bp-maintain-environment.md).
+Now that categories and device groups are established, users of iOS and Android devices enroll their devices, and as they do so, they must choose a category from the list of categories that were configured. Windows users can use the Company Portal website or the Company Portal app to select a category.
+
+1. After enrolling the device go to the [company portal](https://portal.microsoft.com) and choose **My Devices**.
+
+2. Select the enrolled device from the list, and then select a category.
+
+After choosing a category, the device is automatically added to the corresponding group. If a device is already enrolled before you configure categories, the user sees a notification about the device on the Company Portal website. This lets the user know to select a category the next time they access the Company Portal app on iOS/iPadOS or Android.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-device-groups-mdb.md:98]:
```suggestion
After choosing a category, the device is automatically added to the corresponding group. If a device is already enrolled before you configure categories, the user sees a notification about the device on the company portal website. This lets the user know to select a category the next time they access the company portal app on iOS/iPadOS or Android.
```

File: microsoft-365/business-premium/m365bp-review-threats-take-action.md [hunk 3]
```diff
@@ -62,35 +44,37 @@ When you view details about specific threats or devices, you'll see recommendati
...
 
 ## View and manage threat detections in the Microsoft 365 Defender portal
 
-1. Go to the Microsoft 365 Defender portal ([https://security.microsoft.com](https://security.microsoft.com)) and sign in.
+1. Go to the ([Microsoft 365 Defender portal](https://security.microsoft.com)) and sign in.
+
+1. In the navigation pane, choose **Threat Analytics** to see all the current threats. They are categorized by threat severity and type.
 
-2. In the navigation pane, choose **Reports** > **General** > **Security report**.
+1. Click on a threat to see more details about the threat.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-review-threats-take-action.md:55]:
```suggestion
1. Select a threat to see more details about the threat.
```

File: microsoft-365/business-premium/m365bp-review-threats-take-action.md [hunk 4]
```diff
@@ -62,35 +44,37 @@ When you view details about specific threats or devices, you'll see recommendati
...
 
-3. Scroll down to the **Devices** row. If threats were detected on devices, you'll see that information in this row.
+1. In the table, you can filter the alerts according to a number of criteria.
 
-## Manage threat detections in Microsoft Endpoint Manager
+## Manage threat detections in Microsoft InTune
 
-You can use Microsoft Endpoint Manager to manage threat detections. Windows 10 devices must be [enrolled in Intune](/mem/intune/enrollment/windows-enrollment-methods) (part of Microsoft Endpoint Manager).
+You can use Microsoft Endpoint Manager to manage threat detections as well. First, all devices whether Windows, iOS or Android, must be [enrolled in Intune](/mem/intune/enrollment/windows-enrollment-methods) (part of Microsoft Endpoint Manager).
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-review-threats-take-action.md:61]:
```suggestion
You can use Microsoft Endpoint Manager to manage threat detections as well. First, all devices whether Windows, iOS, or Android, must be [enrolled in Intune](/mem/intune/enrollment/windows-enrollment-methods) (part of Microsoft Endpoint Manager).
```

File: microsoft-365/business-premium/m365bp-validate-settings-on-android-or-ios.md [hunk 5]
```diff
@@ -23,13 +23,11 @@ search.appverid:
...
 - MET150
 ms.assetid: f3433b6b-02f7-447f-9d62-306bf03638b0
-description: "Learn how to validate the Microsoft 365 Business Premium app protection settings on your Android or iOS devices. Making security settings for your applications is critical in order to protect the files on your mobile apps and devices."
+description: "Learn how to validate the Microsoft 365 Business Premium app protection settings on your Android or iOS devices. Making security settings for your applications is critical in order to protect the files on your mobile apps and devices from any kind of security threats."
 ---
 
 # Validate app protection settings on Android or iOS devices
 
-> [!NOTE]
-> Microsoft Defender for Business is rolling out to Microsoft 365 Business Premium customers, beginning March 1, 2022. This offering provides additional security features for devices. [Learn more about Defender for Business](../security/defender-business/mdb-overview.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-validate-settings-on-android-or-ios.md:31]:
```suggestion
```

File: microsoft-365/compliance/retention-cmdlets.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+- A retention label policy that you publish needs a rule that defines which labels should be displayed in apps.
+- An auto-apply retention label policy needs a rule that defines the label to apply and the conditions for applying the label.
+
+## Retention cmdlets for most locations
+
+Use the cmdlets in the following table when the locations are **Exchange email**, **SharePoint sites**, **OneDrive accounts**, **Microsoft 365 Groups**, **Skype for Business**, **Exchange public folders**, **Teams chat messages**, or **Teams channel messages**.
+
+Don't use these cmdlets when the locations are for Teams private channel messages, Yammer user messages, or Yammer community messages. These locations have alternative cmdlets that are identified in the [next section](#retention-cmdlets-specific-to-teams-private-channels-and-yammer).
+
+|Cmdlet|Description|Applicable locations|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-cmdlets.md:45]: Please look at the rich diff (preview) - the following two tables do not render correctly. Kindly correct the formatting.

File: microsoft-365/compliance/retention-cmdlets.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+|[Set-RecordReviewNotificationTemplateConfig](/powershell/module/exchange/remove-retentioncompliancepolicy) |Configure the settings for disposition review notification and reminder settings |Exchange email <br /><br /> SharePoint sites <br /><br /> OneDrive accounts <br /><br /> Microsoft 365 Groups|
+|[Set-RetentionCompliancePolicy](/powershell/module/exchange/set-retentioncompliancepolicy)| Configure a policy for retention | Exchange email <br /><br /> SharePoint sites <br /><br /> OneDrive accounts <br /><br /> Microsoft 365 Groups <br /><br /> Skype for Business <br /><br /> Exchange public folders<br /><br /> Teams chat messages <br /><br /> Teams channel messages |
+|[Set-RetentionComplianceRule](/powershell/module/exchange/set-retentioncompliancerule) | Configure settings for retention policies or retention labels |Exchange email <br /><br /> SharePoint sites <br /><br /> OneDrive accounts<br /><br /> Microsoft 365 Groups <br /><br /> Skype for Business <br /><br /> Exchange public folders <br /><br /> Teams chat messages <br /><br /> Teams channel messages |
+
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-cmdlets.md:62]:
```suggestion
```

File: microsoft-365/compliance/retention.md [hunk 8]
```diff
@@ -546,39 +546,9 @@ Retention actions that are logged as auditing events are available only for rete
 
 ## PowerShell cmdlets for retention policies and retention labels
 
-To use the retention cmdlets, you must first [connect to Office 365 Security & Compliance PowerShell](/powershell/exchange/connect-to-scc-powershell). Then, use any of the following cmdlets:
+Use [Office 365 Security & Compliance Center PowerShell](/powershell/exchange/scc-powershell) for Purview retention cmdlets that support configuration at scale, scripting for automation, or might be necessary for advanced configuration scenarios.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention.md:549]:
```suggestion
Use [Office 365 Security & Compliance Center PowerShell](/powershell/exchange/scc-powershell) for Purview retention cmdlets that support configuration at-scale, scripting for automation, or that might be necessary for advanced configuration scenarios.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @opbld15, kindly commit the suggested changes and correct the table formatting as indicated. Many thanks.

**Pull request closed without merging**